static-assets/scripts/main.js

```diff
@@ -747,34 +750,66 @@
         )
       )[1].split(',')[0];
 
+      const max = (
+        (
+          (
+            generalRegExp.match(/\(\?<maxLength>(.*){(.*?)}\)/) ||
+            ['']
+          )[0].match(/{(.*?)}/) ||
+          ['', '']
+        )
+      )[1].split(',')[1];
+
+      const minLength = (
+        (
+          (
+            generalRegExp.match(/\(\?<minMaxLength>(.*){(.*?)}\)/) ||
+            ['']
+          )[0].match(/{(.*?)}/) ||
+          ['', '']
+        )
+      )[1].split(',')[0];
+
+      const maxLength = (
+        (
+          (
+            generalRegExp.match(/\(\?<minMaxLength>(.*){(.*?)}\)/) ||
+            ['']
+          )[0].match(/{(.*?)}/) ||
+          ['', '']
+        )
+      )[1].split(',')[1];
+
       const messages = {
         hasNumbers: formatMessage(passwordRequirementMessages.hasNumbers),
         hasLowercase: formatMessage(passwordRequirementMessages.hasLowercase),
         hasUppercase: formatMessage(passwordRequirementMessages.hasUppercase),
         hasSpecialChars: formatMessage(passwordRequirementMessages.hasSpecialChars, { chars: (allowedChars ? `(${allowedChars})` : '') }),
         noSpaces: formatMessage(passwordRequirementMessages.noSpaces),
         minLength: formatMessage(passwordRequirementMessages.minLength, { min }),
```

Comment:
you renamed var min above so it no longer exists. 
bear in mind the min message prop needs to be called min and the minmax is called minLength

Reply:
create separate vars and regex processing for each. you can’t mix to get min and max separately as they are not guaranteed to be together

Reply:
Hi Roy, 
I'm not sure if I'm understanding this changes good. I didn't remove min and max, I add other variables to support maxMinLength.

Reply:
Got it 👍🏻